contributing.md

```diff
@@ -75,7 +75,7 @@ $ make build
 
 Run the test example:
 ```
-$ pipenv run python examples/carrots.py
+$ pipenv run python examples/carrots_demo/carrots.py
```

Comment:
Rather, can we include this as part of `pytest`?

Reply:
I guess we could. I'm just not sure what was the original intention of putting this script in the contributing.md file? Was it perhaps to let newcomers try and run the code by themselves to get a better feel for the repo? In that case, maybe we should still put this command as part of contributing.md?

Reply:
The idea is new-comers could try the python notebook to run our code. the python files are for how to include them as part of the codebase. 

I just realized this comment was supposed to be for the file in `tests `folder. Like does it run using pytest?

Reply:
@chinmayshah99 I moved jupyter & pandas to devDependencies but the CI seemed to dislike this (?) I encountered jupyter can't be found error (https://github.com/OpenMined/PyDP/runs/1132943621?check_suite_focus=true)